features/messageevent.yml

```diff
@@ -0,0 +1,24 @@
+# BCD: https://github.com/mdn/browser-compat-data/tree/main/api/MessageEvent.json
+# MDN: https://developer.mozilla.org/docs/Web/API/MessageEvent
+# CIU: https://caniuse.com/?search=MessageEvent
+name: MessageEvent
+description: "The MessageEvent interface represents a message received by a target object."
+spec: https://html.spec.whatwg.org/multipage/comms.html#the-messageevent-interface
+#caniuse:
+group: messaging
+#status:
+#  compute_from:
+compat_features:
+  # Already generated in Broadcast channel.
+  # Revisit once the "keys in impl or feature" question is answered.
+  #- api.BroadcastChannel.message_event
+  - api.DedicatedWorkerGlobalScope.message_event
+  - api.MessageEvent
+  - api.MessageEvent.MessageEvent
+  - api.MessageEvent.data
+  - api.MessageEvent.lastEventId
+  - api.MessageEvent.origin
+  - api.MessageEvent.ports
+  - api.MessageEvent.source
```

Comment:
I don't think `MessageEvent` is really a feature. Ideally, we'd site this with _every_ message feature that uses. We can't do that yet, so until then, I think this should live with the oldest feature that uses it. That would be the `window.postMessage` feature below.

Reply:
See note on this at top. Discussed at standup. Lmk if you think we should keep the `messageerror` route as I've done here, or fold into `postMessage`, until we have compound features.

Reply:
Continued at https://github.com/web-platform-dx/web-features/pull/1913#discussion_r1793469067